ferretdb/ferretdb_test.go

```diff
@@ -139,3 +145,34 @@ func Example_tls() {
 
 	// Output: mongodb://127.0.0.1:17028/?tls=true
 }
+
+func TestFerretDB(t *testing.T) {
+	f, err := ferretdb.New(&ferretdb.Config{
+		Listener: ferretdb.ListenerConfig{
+			TCP: "127.0.0.1:0",
+		},
+		Handler:   "sqlite",
+		SQLiteURL: "file:./?mode=memory",
```

Comment:
Let's keep that unit test, but make it use PostgreSQL instead.
And let's remove that integration test that we skip anyway.